services/forms-from-config.ts

```diff
@@ -97,30 +97,49 @@ export const fetchFormDefinitionByUri = async (
 };
 
 const loadConfigForm = async (formName: string) => {
-  const filePath = `${formDirectory}/${formName}/form.ttl`;
-  const metaPath = `${formDirectory}/${formName}/meta.ttl`;
+  const formTtlPath = `${formDirectory}/${formName}/form.ttl`;
+  const metaTtlPath = `${formDirectory}/${formName}/meta.ttl`;
   try {
-    const specification = await fs.readFile(filePath, 'utf-8');
-    const meta = await fs.readFile(metaPath, 'utf-8').catch(() => null);
-    return { formTtl: specification, metaTtl: meta };
+    const formTtl = await fs.readFile(formTtlPath, 'utf-8');
+    const metaTtl = await fs.readFile(metaTtlPath, 'utf-8').catch(() => null);
+    return { formTtl, metaTtl };
   } catch (error) {
     console.error(`Failed to load form ${formName}: ${error}`);
   }
 };
 
 export const loadFormsFromConfig = async () => {
-  const formDirectories = await fetchFormDirectories();
-  formDirectories.forEach(async (formDirectory) => {
-    const form = await loadConfigForm(formDirectory);
-    formsFromConfig[formDirectory] = form;
-    if (!form) {
-      return;
+  const formDirectoryNames = await fetchFormDirectoryNames();
+
+  let hasInvalidForm = false;
+
+  for (const formDirectoryName of formDirectoryNames) {
+    const formDefinition = await loadConfigForm(formDirectoryName);
+    if (!formDefinition) {
+      hasInvalidForm = true;
+      continue;
     }
-    const formUri = await formExtRepo.getFormUri(form.formTtl);
-    formsUriToId[formUri] = formDirectory;
-  });
+    const isValidForm = await comunicaRepo.isValidForm(formDefinition.formTtl);
```

Comment:
Can we use this one  from the submission form helpers addon? Not sure what falls under best effort check ;p
https://github.com/lblod/submission-form-helpers/blob/master/src/validation.js#L13 check it out! Maybe a combination of both is also an option.